src/client/Client.cc

```diff
@@ -1435,6 +1471,10 @@ int Client::verify_reply_trace(int r,
       }
     }
   }
+  if (r >= 0) {
+    assert(target.get());
+    ptarget->swap(target);
+  }
```

Comment:
Why this dance instead of using ptarget directly, as we did previously?
We could still clear it out if there's an error condition.

(I'm just curious about the things that inflate the diff line count; that doesn't mean they're a bad idea but they're harder to digest.)